tests/debugger/oe-gdb/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+add_subdirectory(host)
+add_subdirectory(enc)
+
+add_test(
```

Comment:
Maybe skip this test in simulation mode?

Reply:
Good point. Done.
I will also later work on supporting oe-gdb in simulation mode.